corehq/apps/linked_domain/static/linked_domain/js/domain_links.js

```diff
@@ -139,6 +157,14 @@ hqDefine("linked_domain/js/domain_links", [
         setRMI(initialPageData.reverse('linked_domain:domain_link_rmi'), csrfToken);
 
         var model = DomainLinksViewModel(view_data);
-        $("#domain_links").koApplyBindings(model);
```

Comment:
Curious why this changed, aren't all three tabs still inside the `#domain_links` container?

Reply:
Mix of reasons. I initially started this change using a crud paginated view, and the best way to include that forced me to make this change to avoid applying bindings to the same element twice. Obviously did not continue with that approach, but the idea of separating each tab into its own knockout element seemed ideal. We currently pass in one big view model to #domain_links like you said, but a shift to smaller, more isolated view models would be nice. 

Given that in this PR there aren't any benefits from making this change, I can understand your question. Would love to hear your thoughts if you are opposed to trying to move this direction.

Reply:
Got it. Smaller models sounds good, the big models do get unwieldy, and this page does seem like a good candidate for splitting up.